view/range/index.css

```diff
@@ -13,10 +13,16 @@ body {
     background: var(--opacity-pattern);
     border: none;
   }
+
+  &.is-a::before {
```

Comment:
This rules should be converted to `.range.is-a::before` too

Reply:
I don't understand what's wrong?